### PR TITLE
New package: git-remote-gcrypt-1.3

### PR DESCRIPTION
**Pull request by anjandev**

### CODE REVIEW

File: srcpkgs/git-remote-gcrypt/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'git-remote-gcrypt'
+pkgname=git-remote-gcrypt
+version=1.3
+revision=1
+wrksrc=$pkgname-$version
```
Comment (Chocimier) [inline srcpkgs/git-remote-gcrypt/template:5]: This is default, remove

File: srcpkgs/git-remote-gcrypt/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+build_style=meta
```
Comment (Chocimier) [inline srcpkgs/git-remote-gcrypt/template:6]: meta is for empty packages; just do not set build style

**Comment by anjandev** [comment]
@Chocimier fixed. Thanks for the review.

**Comment by Chocimier** [comment]
When do_install is defined, there is no need for post_install, just put everything into do_.

**Pull request merged**